.gitignore

```diff
@@ -33,5 +33,6 @@ scripts/snicker/joinmarket.cfg
 scripts/snicker/snicker-proposals.txt
 scripts/snicker/candidates.txt
 .qt_for_python/
+.idea/
```

Comment:
This is for some IDE?

Reply:
Yes, PyCharm. Should it be removed?

Reply:
I have no problems with this, if it saves development time. But should be a separate commit.